src/requests/Request.cpp

```diff
@@ -154,6 +160,7 @@ std::ostream& operator<<(std::ostream& os, const Request& request) {
 Request::ParsingErrorException::ParsingErrorException(ErrorType type, const char *error_msg) {
 	_type = type;
     strncpy(_error, "Request parsing exception: ", 27);
+    _error[sizeof(_error) - 1] = '\0';
```

Comment:
You are right, I didn't initialise it properly, however afaik sizeof(_error) - 1 will give the last char.
What you think if we use bzero(_error, 256) ? I will change it if you don't mind